doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -114,7 +114,10 @@ nRF IEEE 802.15.4 radio driver
 Wi-Fi
 -----
 
-|no_changes_yet_note|
+* Added:
+
+  * wifi_credentials library to store credentials
+  * wifi_mgmt_ext library to provide an autoconnect command based on wifi_credentials
```

Comment:
This sounds like we have some new libraries, but where is the actual documentation for them?